### ISSUE TITLE
Allow disabling get_real_pms_url

### ISSUE THREAD

**Issue by neersighted** [Tautulli/Tautulli#1133]
As I run my plex server in a docker container with private networking, `get_real_pms_url` ends up doing the wrong thing and breaks resolution. I would like to disable all URL-resolution functionality, and instead have a hardcoded URL. PlexPy does the right thing for most users, but with my unusual network topology there is no way to make it work correctly (IPs may change, so DNS lookup is required).

I'd also like the ability to disable SSL verification, though that is not as high a priority.

Thanks!

**Comment by neersighted** [comment]
I also see now that this would require disabling SSL verification. That would be nice to have as well. :smile:

**Comment by JonnyWong16** [comment]
If you uncheck "Use SSL", then it will use whatever IP address you specify.

You can disable SSL verification by setting `verify_ssl_cert = 0` in the config.ini file. (I think)

**Comment by neersighted** [comment]
@JonnyWong16 From skimming the code, it appears that there is no way to use HTTPS and a fixed IP at the same time. In addition, I'd like to use a hostname, and not an IP. Is there any way to do that currently?

**Comment by JonnyWong16** [comment]
No HTTPS with a fixed IP. No hostname.

File this as a feature request.
https://github.com/JonnyWong16/plexpy#feature-requests

**Comment by silverbullettruck2001** [comment]
@JonnyWong16 is this a new requirement?  I have used PlexPy for the last 9 months and I have always required SSL on my Plex server and I have always used a fixed IP address.  It hasn't stopped connecting to my Plex server until just recently.  I have submitted a feature request here: https://feathub.com/JonnyWong16/plexpy/+236

If this could be fixed soon I would appreciate it as this won't connect to Plex anymore for me.
